M2/INSTALL

```diff
@@ -182,10 +182,24 @@ various modern operating systems:
       a tar file that can be untarred anywhere and renamed.)
     Install the programs and libraries needed for building:
 	brew update
-	brew install --build-bottle pkg-config autoconf automake gnu-tar libtool boost libmpc xz wget
-	brew install --build-bottle bdw-gc libatomic_ops gdbm glpk yasm make
+	brew install --build-bottle pkg-config autoconf automake gnu-tar libtool boost libmpc xz wget libomp
+	brew install --build-bottle bdw-gc libatomic_ops gdbm glpk yasm make cmake
       You may omit the option "--build-bottle" if you will not be distributing your Macaulay2
       binaries to other people with CPU's that may not have the same capabilities as yours.
+    Starting with M2 version 1.12.0.1, (i.e. 1.13), the apple clang c++ compiler has some annoying
```

Comment:
We should try to make this work with the native compilers provided with Mac OS X.